handlers/admin/adminDLQPage.go

```diff
@@ -106,7 +106,7 @@ func AdminDLQPage(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
-	handlers.TemplateHandler(w, r, "admin/dlqPage.gohtml", data)
+	handlers.TemplateHandler(w, r, "dlqPage.gohtml", data)
```

Comment:
@jules, this page must be in `admin/` as it's admin move the page rather than change this if required.